client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -194,6 +196,7 @@ class CalypsoifyIframe extends Component<
 			this.iframeRef.current &&
 			this.iframeRef.current.contentWindow
 		) {
+			window.performance.mark( 'iframe_loaded' );
```

Comment:
Is this intended to happen for each iframe event or just for the initial load? If it's the initial load, it might make sense to add this to the beginning of the `onIframeLoaded` method instead.